lib/toggl/TogglService.ts

```diff
@@ -278,11 +291,18 @@ export default class TogglService {
         )}...`;
       }
       const duration = this.getTimerDuration(this._currentTimeEntry);
-      const minutes = Math.floor(duration / 60);
-      const time_string = `${minutes} minute${minutes != 1 ? "s" : ""}`;
+      // @ts-expect-error
```

Comment:
Same remark about the `ts-expect-error` as above.

Reply:
Resolved as above!